Signal/src/ViewControllers/SettingsTableViewController.m

```diff
@@ -78,153 +51,205 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder
+                                              actionBlock:^{
+                                                  [weakSelf showInviteFlow];
+                                              }]];
+    [section addItem:[OWSTableItem disclosureItemWithText:NSLocalizedString(@"SETTINGS_PRIVACY_TITLE",
+                                                              @"Settings table view cell label")
+                                              actionBlock:^{
+                                                  [weakSelf showPrivacy];
+                                              }]];
+    [section addItem:[OWSTableItem disclosureItemWithText:NSLocalizedString(@"SETTINGS_NOTIFICATIONS", nil)
+                                              actionBlock:^{
+                                                  [weakSelf showNotifications];
+                                              }]];
+    [section addItem:[OWSTableItem disclosureItemWithText:NSLocalizedString(@"LINKED_DEVICES_TITLE",
+                                                              @"Menu item and navbar title for the device manager")
+                                              actionBlock:^{
+                                                  [weakSelf showLinkedDevices];
+                                              }]];
+    [section addItem:[OWSTableItem disclosureItemWithText:NSLocalizedString(@"SETTINGS_ADVANCED_TITLE", @"")
+                                              actionBlock:^{
+                                                  [weakSelf showAdvanced];
+                                              }]];
+    [section addItem:[OWSTableItem disclosureItemWithText:NSLocalizedString(@"SETTINGS_ABOUT", @"")
+                                              actionBlock:^{
+                                                  [weakSelf showAbout];
+                                              }]];
+
+    [section addItem:[OWSTableItem itemWithCustomCellBlock:^{
+        UITableViewCell *cell = [UITableViewCell new];
+        UIButton *button = [UIButton buttonWithType:UIButtonTypeCustom];
+        button.backgroundColor = [UIColor ows_destructiveRedColor];
+        [button setTitleColor:[UIColor whiteColor] forState:UIControlStateNormal];
+        [button setTitle:NSLocalizedString(@"SETTINGS_DELETE_ACCOUNT_BUTTON", @"") forState:UIControlStateNormal];
+        button.titleLabel.font = [UIFont ows_mediumFontWithSize:18.f];
+        button.titleLabel.textAlignment = NSTextAlignmentCenter;
+        [cell.contentView addSubview:button];
+        [button autoSetDimension:ALDimensionHeight toSize:50.f];
+        [button autoVCenterInSuperview];
+        [button autoPinEdgeToSuperviewEdge:ALEdgeLeft
+                                 withInset:cell.layoutMargins.left + cell.contentView.layoutMargins.left];
+        [button autoPinEdgeToSuperviewEdge:ALEdgeRight
+                                 withInset:cell.layoutMargins.right + cell.contentView.layoutMargins.right];
```

Comment:
I finally learned the right way to align custom cell content with the default cell content.